docs/src/main/asciidoc/kubernetes-client.adoc

```diff
@@ -138,3 +138,85 @@ Note that to take advantage of these features, the `quarkus-test-kubernetes-clie
         <scope>test</scope>
     </dependency>
 ----
+
+== Note on implementing the Watcher interface
+
+Due to the restrictions posed by GraalVM, extra care needs to be taken when implementing a `io.fabric8.kubernetes.client.Watcher` if the application is intended to work in native mode.
```

Comment:
```suggestion
Due to the restrictions imposed by GraalVM, extra care needs to be taken when implementing a `io.fabric8.kubernetes.client.Watcher` if the application is intended to work in native mode.
```
?

Reply:
Yeah, that's better I think, I'll update